### DISCUSSION TITLE
中文括号在win下面很丑（mac下面好看多了）

### DISCUSSION THREAD

**Discussion by wany16** [tuna/thuthesis#738] [Q&A] [answered]
同样的中文括号（非英文括号），在win和mac下编译都没有问题，但是显示出来的效
果不一样，win下面丑多了。
[mac-中文括号.pdf](https://github.com/tuna/thuthesis/files/8245325/mac-.pdf)
[win-中文括号.pdf](https://github.com/tuna/thuthesis/files/8245326/win-.pdf)

**Comment by zepinglee** [comment, marked as answer]
在 macOS 编译会调用系统自带的华文宋体，而在 Windows 编译会调用中易宋体，两者的括号字形不一样，属于正常现象。

另外提醒一下，根据 CY/T 154—2017《中文出版物夹用英文的编辑标准》，这里的逗号应该使用英文逗号。
<img width="696" alt="Screen Shot 2022-03-14 at 21 54 54" src="https://user-images.githubusercontent.com/12290822/158186525-11692187-bcdf-4fd4-b002-fb97be952129.png">
